CSharp/BatchExplorer/Models/TaskModel.cs

```diff
@@ -279,6 +267,18 @@ public override List<PropertyModel> PropertyModel
                         //Swallow the exception to stop popups from occuring for every bad VM
                     }
 
+                    IPagedEnumerable<SubtaskInformation> subtasks = this.Task.ListSubtasks(OptionsModel.Instance.ListDetailLevel);
```

Comment:
No, because "Bad VMs" won't cause an issue with ListSubtasks since it doesn't actually talk to a VM.  As opposed to ListFiles which does.